app/experimenter/outcomes/__init__.py

```diff
@@ -0,0 +1,57 @@
+import os
+from dataclasses import dataclass
+
+import toml
+from django.conf import settings
+
+from experimenter.experiments.constants import NimbusConstants
+
+
+@dataclass
+class Outcome:
+    app_id: str
+    app_name: str
+    description: str
+    friendly_name: str
+    slug: str
+
+
+class Outcomes:
+    _outcomes = None
+
+    @classmethod
+    def _load_outcomes(cls):
+        outcomes = []
+
+        for app_name in os.listdir(settings.JETSTREAM_CONFIG_OUTCOMES_PATH):
+            app_path = os.path.join(settings.JETSTREAM_CONFIG_OUTCOMES_PATH, app_name)
+
+            for outcome_name in os.listdir(app_path):
+                outcome_path = os.path.join(app_path, outcome_name)
+
+                with open(outcome_path, "r") as outcome_file:
+                    outcome_toml = outcome_file.read()
+                    outcome_data = toml.loads(outcome_toml)
+
+                    outcomes.append(
+                        Outcome(
+                            app_id=NimbusConstants.APP_NAME_APPLICATION[app_name],
```

Comment:
If we support a new app in jetstream-config this will break the Experimenter build -- what about ignoring paths for unrecognized app_names?

Reply:
We could ignore it or we could break on it and be forced to add it to our list?  Will there be jetstream outcome apps that experimenter will never know about?

Reply:
> Will there be jetstream outcome apps that experimenter will never know about?

I agree not "never" but maybe transiently.

Hm, I don't think tests here run against the live data so I think the failure mode is actually that the container would build fine but the app would crash on load.

Reply:
Yep, which would get caught at CI time, the container would fail to start during the integration tests.  I'm fine with doing either, ignoring apps that aren't recognized or breaking entirely, which do you prefer?